samples/iolib/src/main/cpp/player/SimpleMultiPlayer.cpp

```diff
@@ -143,7 +145,7 @@ bool SimpleMultiPlayer::startStream() {
     return false;
 }
 
-void SimpleMultiPlayer::setupAudioStream(int32_t channelCount) {
+void SimpleMultiPlayer::setupAudioStream(int32_t channelCount, oboe::PerformanceMode performanceMode) {
     __android_log_print(ANDROID_LOG_INFO, TAG, "setupAudioStream()");
     mChannelCount = channelCount;
 
```

Comment:
Shouldn't the performance mode be used when calling `openStream`?

Reply:
Fixed